requirements.txt

```diff
@@ -1,7 +1,9 @@
+.
```

Comment:
By adding `.` to requirements.txt, we need to update Dockerfile to `cd /src` before calling `pip install -r requirementst.txt` to make sure the path resolves